confluent-docker-utils.yaml

```diff
@@ -37,6 +37,8 @@ pipeline:
       # Workaround to fix `AttributeError: cython_sources` issue
       # https://github.com/yaml/pyyaml/issues/724#issuecomment-1638587228
       sed -i 's|PyYAML~=5.4.1|PyYAML==6.0.1|g' requirements.txt
+      # Fix CVE GHSA-h4gh-qq45-vh27
+      sed -i 's|cryptography~=42.0.4|cryptography==43.0.1|g' requirements.txt
```

Comment:
What about using `~=43.0.1` ?

Reply:
Thanks for pointing this, yes we shiiould use ~=, will do that so that it can pick up updates, will do it

Reply:
fixed